src/app/todo-list/todo-list.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component } from '@angular/core';
+
+import { Todo } from '../../models/todo.model';
+
+@Component({
+  selector: 'app-todo-list',
+  templateUrl: './todo-list.component.html',
+  styleUrls: ['./todo-list.component.scss']
+})
+export class TodoListComponent {
+  editingTodoId: string | null = null;
+
+  todos: Todo[] = [
+    { id: '1', title: 'Install Angular', status: 'done'},
```

Comment:
I m just wondering, why would and id be a string and not a number?

Reply:
in the future I plan to add firebase and ID will be a string there